src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
According to the checklist, class and interface names should not be verbs. The name 'AreaCalculator' suggests an action rather than an entity. Consider renaming it to something that represents an entity, such as 'AreaComputable', 'AreaProvider', or another suitable noun that follows Java naming conventions.